R/type-github.R

```diff
@@ -11,6 +11,10 @@
 parse_remote.remote_specs_github <- function(specs, config, ...) {
 
   pds <- re_match(specs, github_rx())
+  if (is.na(pds$.match)) {
+    pds <- re_match(specs, github_url_rx())
+    pds$subdir <- ""
```

Comment:
The URL syntax does not currently support subdirectories, not sure if we want to add or not. For now I just set it to "" in this case.

Reply:
OK.